packages/ffe-account-selector-react/src/components/account-selector/AccountSelector.js

```diff
@@ -60,24 +65,34 @@ const AccountSelector = ({
         }
     };
 
+    const onBlur = () => {
+        if (allowCustomAccount) {
+            onAccountSelected({
+                name: inputValue,
+                accountNumber: inputValue,
+            });
+        }
+        if (inputProps && inputProps.onBlur) {
```

Comment:
Det var vel ok å skrive ```if (inputProps?.onBlur)``` nå for tiden, eller?

Reply:
Sant! Fikser det samme i sjekken av onChange også :ok_hand: